<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GdalToolsWidget</class>
 <widget class="QWidget" name="GdalToolsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>125</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Near Black</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Input file</string>
       </property>
       <property name="buddy">
        <cstring>inSelector</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Output file</string>
       </property>
       <property name="buddy">
        <cstring>outSelector</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="nearCheck">
       <property name="text">
        <string>How &amp;far from black (or white)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="nearSpin">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="GdalToolsInOutSelector" name="inSelector" native="true"/>
     </item>
     <item row="1" column="1">
      <widget class="GdalToolsInOutSelector" name="outSelector" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="whiteCheckBox">
     <property name="text">
      <string>Search for nearly &amp;white (255) pixels instead of black ones</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GdalToolsInOutSelector</class>
   <extends>QWidget</extends>
   <header>.inOutSelector</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
